tsconfig.cjs.build.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "declaration": false,
+    "outDir": "dist",
+    "module": "CommonJS"
```

Comment:
with the new set of three `tsconfig.json` files, looks like tests are now emitted to `dist/` where they weren't before. i'd prefer to keep them out of builds if possible!

either the `exclude` that's in `tsconfig.build.json` needs to be added to `tsconfig.cjs.build.json`, or `tsconfig.cjs.build.json` needs to extend `tsconfig.build.json` instead of `tsconfig.json`, or we need to go back to excluding test files in the root tsconfig and explicitly including them in a `tsconfig.eslint.json` as before.

i think i'd prefer that last option since it's the least disruptive and makes it clearer that the `tsconfig.eslint.json` is an exception for a different piece of tooling. is there a reason you changed things around like this?

Reply:
Thank you for spotting this, I extended indeed the wrong file. I adjusted it in the last commit.